server/athenian/api/controllers/user_controller.py

```diff
@@ -167,7 +166,9 @@ async def set_account_features(request: AthenianWebRequest, id: int, body: dict)
         for i, feature in enumerate(features):
             if feature.name == DBAccount.expires_at.name:
                 try:
-                    expires_at = deserialize_datetime(feature.parameters)
+                    expires_at = deserialize_datetime(
+                        feature.parameters, min_=None, max_future=None,
```

Comment:
I would leave the default minimum here because I've seen the client sending us weird dates Before Christ which we don't want to save in the DB.

Reply:
done